src/main/scala/io/iohk/ethereum/consensus/atomixraft/Miner.scala

```diff
@@ -0,0 +1,162 @@
+package io.iohk.ethereum.consensus
+package atomixraft
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import akka.pattern.ask
+import akka.util.Timeout
+import io.iohk.ethereum.blockchain.sync.RegularSync
+import io.iohk.ethereum.consensus.atomixraft.Miner._
+import io.iohk.ethereum.domain.{Block, Blockchain}
+import io.iohk.ethereum.jsonrpc.EthService
+import io.iohk.ethereum.mining.{BlockGenerator, PendingBlock}
+import io.iohk.ethereum.nodebuilder.Node
+import io.iohk.ethereum.transactions.PendingTransactionsManager
+import io.iohk.ethereum.transactions.PendingTransactionsManager.PendingTransactionsResponse
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.{Failure, Success}
+
+class Miner(
+  blockchain: Blockchain,
+  blockGenerator: BlockGenerator,
+  pendingTransactionsManager: ActorRef,
+  syncController: ActorRef,
+  config: AtomixRaftConfig,
+  ethService: EthService,
```

Comment:
it doesn't seem to be needed here

Reply:
Indeed it is [gone](https://github.com/input-output-hk/mantis/blob/feature/consensus_as_a_service_EC458/src/main/scala/io/iohk/ethereum/consensus/atomixraft/AtomixRaftMiner.scala#L21..L26) now.